### DISCUSSION TITLE
Temporal numerical targets

### DISCUSSION THREAD

**Discussion by rzarai** [iSoron/uhabits#1124] [Feature Requests]
Currently we are having just numerical targets, which is awesome feature, but the very first thing I wanted to track with it is an amount of time, and it doesn't bring option to track in temporal units, such as hours, minutes, seconds. Would be super awesome to add time tracking as well.

**Comment by KristianTashkov** [comment]
I know it's not the same, but currently for temporal tracking I'm just using numerical habit with a unit of `minutes`. Beside the "neat" formatting of hours it still gets the job done!

**Reply by rzarai** [reply]
Yep. The same is performed by me. But still it'd be cool to have temporal units available.

**Comment by herrbrixter** [comment]
Duplicate of #953
